frontend/app/routes/protected/person-case/primary-docs.tsx

```diff
@@ -85,43 +123,13 @@ export default function PrimaryDocs({ loaderData, actionData, params }: Route.Co
   const fetcherKey = useId();
   const fetcher = useFetcher<Info['actionData']>({ key: fetcherKey });
   const isSubmitting = fetcher.state !== 'idle';
-  const errors = fetcher.data?.errors;
+  const errors: { currentStatusInCanada?: string[]; documentType?: string[] } | undefined = fetcher.data?.errors ?? undefined;
```

Comment:
The `?? undefined` is unnecessary here.

Reply:
Why `errors` variable is typed with `: { currentStatusInCanada?: string[]; documentType?: string[] } | undefined`? This should work.

```tsx
const errors = fetcher.data?.errors;
```